src/app/views/settings/settings.controller.coffee

```diff
@@ -0,0 +1,57 @@
+@App.controller 'SettingsController', ($timeout, $window, CONFIG_JSON_SCHEMA, MnoeTenant) ->
+  'ngInject'
+  vm = this
+
+  # vm.isLoading = false
+
+  vm.settingsModel = {}
+  vm.settingsSchema = CONFIG_JSON_SCHEMA
+  vm.settingsForm = [
+    "*",
+    {
+      type: "submit"
+      title: "Save"
+    }
+  ]
+
+#  vm.settingsForm = [
+#    {
+#      type: "tabs"
+#      tabs: [
+#        {
+#          title: "Admin Panel"
+#          items: ["admin_panel"]
+#        }
+#        {
+#          title: "Dashboard"
+#          items: ["*"]
+#        }
+#      ]
+#    }
+#    {
+#      type: "actions"
+#      items: [
+#        type: "submit"
+#        title: "Save"
+#      ]
+#    }
+#  ]
+
+  MnoeTenant.get().then(
+    (response) ->
+      vm.settingsModel = response.data.frontend_config
+  )
+
+  vm.saveSettings = (form) ->
+#    vm.isLoading = true
+    MnoeTenant.update(vm.settingsModel).then(
+      ->
+        $timeout(->
```

Comment:
I'll need to find a better way to wait the configuration propagation. Let's discuss it.